sysutils/pkg/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        freebsd pkg 1.10.4
+categories          sysutils archivers
+platforms           darwin
+license             BSD
+maintainers         {@apowers313 ato.ms:apowers} \
+                    openmaintainer
+description         Package management tool for FreeBSD
+long_description    Pkg is the Next Generation package management tool for FreeBSD. Its main goals are to faciliate remote binary package upgrades. It also works with ports without remote binary packages.
+homepage            https://wiki.freebsd.org/pkgng
+
+depends_lib         port:libarchive \
+                    port:openssl
+
+checksums           sha256 ced2627aae44b95a752565f2bc067888a6f4f3c2e856c233fc1567016788f6cb \
+                    rmd160 b30e8e926fa44612347bfcd81f1896bb44e48198 \
+                    size 3398284
+
+test.run            yes
+
+pre-configure {
+    if {![file exists "${worksrcpath}/configure"]} {
+        system -W ${worksrcpath} ./autogen.sh
+    }
+}
+
+configure.pre_args-append --mandir=${prefix}/share/man
```

Comment:
Any reason you're using `configure.pre_args` over `configure.args` here? We usually only put `--prefix` in `pre_args`.

Reply:
Same as my previous comment -- I was following the convention I found in other packages.

Reply:
It doesn't particularly matter if you put configure arguments in `configure.pre_args`, `configure.args`, or `configure.post_args`—they all get added, in that order—but unless you're dealing with a very complicated portfile or build system where you need to specify configure arguments from a variety of different places in your portfile and the order of the arguments is significant, you should follow our conventions which are that the argument that specifies the prefix goes in `configure.pre_args` and everything else goes in `configure.args`. I'm sure we have a few ports where `configure.pre_args` is used for arguments unrelated to the prefix, and in most cases that's a stylistic faux pas that should be corrected.